testsuite/benchmark-workloads/package-generator/src/main.rs

```diff
@@ -74,6 +76,7 @@ fn main() -> anyhow::Result<()> {
     create_prebuilt_packages_bundle(
         args.prebuilt_packages_file_dir,
         all_package_paths,
+        args.experiments,
```

Comment:
This breaks external projects, we should pass this via config (I hope we have a public constructor🤣) and just extend to one with experiments and update that code later?

Reply:
Seems like we don't have it: so it does break things a bit: can you add one?

Reply:
done